MODULE.bazel

```diff
@@ -13,6 +13,14 @@ bazel_dep(name = "bazel_skylib", version = "1.4.2")
 bazel_dep(name = "rules_python", version = "0.29.0")
 bazel_dep(name = "platforms", version = "0.0.7")
 
+
+# Custom python version for testing only
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    is_default = False,
```

Comment:
I was expecting this one to be explicitly named something ... I guess that's not how rules_python works eh?

Reply:
this directly from their example, i don't see any name attribute, so no 🤷🏻‍♂️

Reply:
Yeah they must have their own scheme for avoiding naming collisions in the external/ folder and also for referencing the version. Okay...